salz-core/src/lib/Map.hs

```diff
@@ -21,48 +21,48 @@ mapSize = 100
 -- MInt
 --------------------------------------------------------------------------------
 
-data MInt = I Integer deriving Eq
+data MInt = MInt Integer deriving Eq
```

Comment:
I have a feeling `MInt` is something like modulo'd integer? I seem to be reading a lot of C# nowadays, and the class names, variable names, etc are painfully verbose. But at least fairly understandable. Since we don't do much haskell, maybe be a little verbose with your naming?

Reply:
For `MInt`, as per the comment [here](https://github.com/Lugarun/salz/pull/36#discussion_r411396570), you’re right on the money.

Reply:
Yeah, `MInt` is something like that. I can change that variable name aswell